client/src/components/Modals/ModalDocument.vue

```diff
@@ -26,6 +26,16 @@
       </div>
       <similar-docs :doi="doi" class="m-3"/>
     </div>
+    <button
+      v-if="linkToKnowledgeSpace"
+      slot="footer"
+      type="button"
+      class="btn btn-link"
+      @click="openKnowledgeView()"
```

Comment:
It would be good to pass the `doi` to the `docsCards` view so we can add an automatic query there for that paper. However, I don't think we have the DOI pill in that space.

Reply:
That make sense, I thought of that, but there is no easy way to connect the route to LexBar.
I added the DOI to the Route query nonetheless.

https://user-images.githubusercontent.com/636801/125306460-3cd30e00-e2fd-11eb-9962-18fa66f059da.mov